native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/postgresql-native/reflect-config.json

```diff
@@ -256,19 +217,103 @@
     "methods": [
       {
         "name": "<init>",
-        "parameterTypes": []
+        "parameterTypes": [ ]
       }
     ]
   },
   {
```

Comment:
How are we handling test case related configurations? @TharmiganK

Reply:
Test related configurations can be added to the `test-util` if we have one in the module. In this case, we don't have a `test-util` then  we could add a separate jar which only consists the native-image configs (i.e. `META-INF/native-image/<needed-config-files>`), to the `Ballerina.toml` file with `testOnly` scope.
We could use this as a workaround until tooling supports adding additional native-image configs through some config files.